src/cascadia/TerminalSettingsModel/CascadiaSettings.cpp

```diff
@@ -103,7 +103,7 @@ Model::CascadiaSettings CascadiaSettings::Copy() const
             for (const auto& profile : targetProfiles)
             {
                 allProfiles.emplace_back(*profile);
-                if (!profile->Hidden())
+                if (!profile->Hidden() && !profile->Orphaned())

```

Comment:
this one is not commented because it does not differ from the contents of 18188.